packages/editor-ui/src/components/BecomeTemplateCreatorCta/BecomeTemplateCreatorCta.vue

```diff
@@ -0,0 +1,78 @@
+<script setup lang="ts">
```

Comment:
Subjective - My eyesight is not the best and I find the font size in this component too tiny.